.gitignore

```diff
@@ -16,4 +16,4 @@ vendor/
 junit.xml
 Makefile.dapper
 Dockerfile.*
-
+.*.swp
```

Comment:
Please [configure your local gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) instead of adding environment-specific entries to the project’s `.gitignore`.